extensions/wikia/FounderEmails/events/FounderEmailsViewsDigestEvent.class.php

```diff
@@ -27,11 +27,16 @@ public function process ( Array $events ) {
 
 		$wgTitle = Title::newMainPage();
 		// Get list of founders with digest mode turned on
-		$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-views-digest' );
+		if ( empty( $events ) ) {
+			$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		} else {
+			$cityList = $events;
```

Comment:
@jsutterfield, oh I didn't notice that, I'll have to change the comment.  The issue is with the ```maintenance.php``` script in the root directory of this extension.  This script takes a ```--wikiId``` argument which is intended to make it run *only* against that wiki.  For the events that do take an ```$events``` array, that array is populated by ```maintenance.php``` and restriction to just the wiki ID given is enforced.  However the events that are not given an ```$events``` array gather their own data and run on every single wiki.

This means that if you run the maintenance script for these types of events with only one wiki ID given, you actually run it for all wikis (as I did when I was testing this).